submissions/Harnytskyi/html-movie-seat-booking/index.html

```diff
@@ -0,0 +1,635 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Cinema Ticket Booking</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" type="text/css" media="screen" href="css/style.css" />
+</head>
+
+<body>
+    <main>
+        <div class="booking_block">
+            <div id="booking_wrapper" class="booking_wrapper">
+                <div class="hall_container">
+                    <div class="hall">
+                        <div class="block_item"></div>
```

Comment:
Avoid using empty blocks for defining space. CSS grid can handle this without empty blocks. [Read more](https://gridbyexample.com/examples/example8/)

Reply:
I defined space into columns without empty blocks, but I have not idea how I can skip the whole row.

Reply:
You can use `grid-areas` [Link](https://gridbyexample.com/examples/example11/) or Line-based placement named lines with spans [Link](https://gridbyexample.com/examples/example8/)